src/RefreshManager.m

```diff
@@ -33,6 +33,7 @@
 #import "NSNotificationAdditions.h"
 #import "VTPG_Common.h"
 #import "FeedItem.h"
+#import <QuartzCore/QuartzCore.h>
```

Comment:
I am not exactly adding UI, I am trying to fix the current one.
I noticed like @Eitot that problem seems [related to existing calls to ActivityItem](https://github.com/ViennaRSS/vienna-rss/issues/833#issuecomment-287895810), but from my tests, his suggestion was not sufficient and I needed the trick [found here](http://stackoverflow.com/questions/12297787/coreanimation-warning-deleted-thread-with-uncommitted-catransaction).